docs/source/technical_tutorials/visualization/stac_ipyleaflet.ipynb

```diff
@@ -0,0 +1,208 @@
+{
```

Comment:
<p>Any chance to get those coordinates into a code block or variable within the notebook? I know it was a recent ask. We can update this section once that feature is working.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/190/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
I'll add this to the queue - although I'll need to revisit next week after the current to dos.